### PR TITLE
Tweaks to hiring document

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,17 +60,20 @@ You can retrieve the list of all tracks from: https://exercism.org/api/v2/tracks
...
-- `exercise`: Pass a partial exercise name (e.g. `Fer`) to filter just exercises with the work "Fer" in.
+You can use the following params to modify which records are returned:
+
+- `page`: Specify to return different pages of the results (first page has page number `1`).
+- `track`: Pass a complete track slug (e.g. `python`) to only return the Ruby track's testimonials.
+- `exercise`: Pass a partial exercise name (e.g. `ming`) to only return the exercises that contain the word "ming".
 - `order`: Specify either `newest_first` or `oldest_first` (default) to switch the order.
 
-A complete URL might be `https://exercism.org/api/v2/hiring/testimonials?page=2&track=javascript&exercise=bob&order=newest_first`.
+A complete URL might be `https://exercism.org/api/v2/hiring/testimonials?page=2&track=python&exercise=ming&order=newest_first`.
```
Comment (ErikSchierboom) [inline README.md:72]: The previous url didn't return any results. This returns just 23 results, but still has multiple pages

File: README.md [hunk 2]
```diff
@@ -18,30 +21,32 @@ A key criteria for this job role is how accurately you can replicate designs.
...
+Ideally, you'll use Tailwind 3 for the CSS, but plain CSS is totally fine too.
 
 ## Functionality
 
-Functionality should be written in React. 
-Ideally you'll use Typescript, but JavaScript is fine too.
+The functionality should be implemented using React.
+Ideally, you'll use TypeScript, but JavaScript is fine too.
 
 Please provide some tests as well.
```
[the remark this answers is not in the document]
Comment (iHiD) [reply README.md:31]: yes!

File: README.md [hunk 3]
```diff
@@ -55,17 +60,20 @@ You can retrieve the list of all tracks from: https://exercism.org/api/v2/tracks
...
+
+The endpoint returns JSON with two top-level keys:
 
-The endpoint returns JSON with three keys:
-- `results`: The testimonials containing the relevant information to render on the UI.
-- `pagination`: Pagination data to allow you to render the pagination section.
+- `testimonials`: An object containing information related to the page of testimonials that is to be rendered.
+  - `results`: The testimonials containing the relevant information to render on the UI.
+  - `pagination`: Pagination data to allow you to render the pagination section.
 - `tracks`: A list of all tracks that this user has given testimonials on.
```
Comment (iHiD) [inline README.md:80]:
```suggestion
  - `tracks`: A list of all tracks that this user has given testimonials on.
```

Comment (iHiD) [reply README.md:80]: This is inside testimonials too I think.

**Pull request merged**